cpp/tests/utilities/column_utilities.cu

```diff
@@ -175,6 +414,8 @@ class corresponding_rows_not_equivalent {
 std::string stringify_column_differences(cudf::device_span<int const> differences,
                                          column_view const& lhs,
                                          column_view const& rhs,
+                                         column_view const& lhs_row_indices,
+                                         column_view const& rhs_row_indices,
                                          bool print_all_differences,
```

Comment:
Are you considering switching `print_all_differences` to an `enum print_level { ALL, FIRST, NONE }`?

Reply:
Centralizing all these threads to here:
https://github.com/rapidsai/cudf/pull/8666#issuecomment-881765612